### ISSUE TITLE
[Accessibility][CreateDashboardCenturion] Required form controls do not convey or expose the fact that they are required

### ISSUE THREAD

**Issue by mnischay** [awslabs/iot-application#1815]
### Description 

CreateDashboardCenturion : Required form controls do not convey or expose the fact that they are required

 Target Count: 2 
 Standard: 3.3.2 Labels or Instructions (A)

### AC

- [ ] The form controls should be AAA compliant i.e 3.3.2 Labels or Instructions (A) should be fixed in the create dashboard page

### Notes and Advice 
Users are required to input data before a form can be completed, but the required fields are not indicated.
 
Name
Description
 Recommendation 
Ensure required fields indicate their required state to users. This involves taking the following steps:
 
Indicate a field is required by including the text (required) or an asterisk * in the label of the field
If an asterisk is used:
Include a message at the start of the form to explain the meaning of the asterisk, or
Hide it with an aria-hidden="true" attribute to ensure that screen readers do not announce the ambiguous "star" text when the associated field receives focus.
For screen reader users:
Apply the aria-required attribute to the <input> and set its value to true. This will ensure that the <input> is exposed to assistive technologies as a required field, and thus screen readers will announce this fact to their users (e.g. by announcing "required" after announcing the label and any contents within the field).
 
Example markup for a required field
 
<label for="employeeName">Employee <span aria-hidden="true">(required)</span></label>
<input type="text" id="employeeName" aria-required="true">
 
Another possible example markup for a required field
 
<form>
  <p>Fields with an asterisk * are required</p>
  ...
  <label for="employeeName">Employee *</label>
  <input type="text" id="employeeName" aria-required="true">
  ...
</form>
 
In this example, it is not necessary to hide the asterisk as users have been advised what it means. ^^^
